### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by omahs**
## Type of change
- Documentation (typos)

### CODE REVIEW

File: .docs/contributing-book/src/documentation/project-quality/documentation/specification.md [hunk 1]
```diff
@@ -18,4 +18,4 @@ A technical specification is aimed at users that may be regarded as experts / kn
 
 <br> 
 
-> **Note:** Diagrams are a fantastic visual aid no matter the level of detail
\ No newline at end of file
+> **Note:** Diagrams are a fantastic visual aid no matter the level of detail
```
Comment (SwayStar123) [inline .docs/contributing-book/src/documentation/project-quality/documentation/specification.md:21]: Whats changing here?

**Review by bitzoic — APPROVED** [review]
Thanks for contributing!

File: .docs/contributing-book/src/documentation/project-quality/documentation/specification.md (hunk 1 above)
Comment (Braqzen) [reply .docs/contributing-book/src/documentation/project-quality/documentation/specification.md:21]: Seems like git is being strange

Comment (Braqzen) [reply .docs/contributing-book/src/documentation/project-quality/documentation/specification.md:21]: Seems like git is being strange

**Review by Braqzen — APPROVED** [review]

**Pull request merged**